flake.nix

```
@@ -152,6 +153,89 @@
               ];
               treefmt = pkgs.treefmt;
             };
+            hydra-node = makeTest {
+              name = "hydra-node";
+              nodes = {
+                hydraNode = { ... }: {
+                  environment.systemPackages = [
+                    pkgs.cardano-node
+                    pkgs.cardano-cli
+                    pkgs.check-jsonschema
+                  ];
+                  networking.firewall.enable = false;
+                  imports = [ ];
+                  virtualisation = {
+                    cores = 16;
+                    memorySize = 16384;
+                  };
+                };
+              };
+
+              testScript = ''
+                hydraNode.wait_for_unit("multi-user.target")
+                hydraNode.succeed("cp ${self}/hydra-node/golden . -r")
+                hydraNode.succeed("cp ${self}/hydra-node/json-schemas . -r")
+                hydraNode.succeed("mkdir -p test")
+                hydraNode.succeed("cp ${self}/hydra-node/test/tls test/ -r")
+                hydraNode.succeed("${hsPkgs.hydra-node.components.exes.test-exe}/bin/test-exe")
```

Must: not require additional `.cabal` components to run this

The test-suite _is_ already a binary you can run so this would be something like:
```suggestion
                hydraNode.succeed("${hsPkgs.hydra-node.components.tests.tests}/bin/tests")
```